docs/source/common/optimizers.rst

```diff
@@ -300,7 +304,7 @@ override the :meth:`optimizer_step` function.
 
 For example, here step optimizer A every 2 batches and optimizer B every 4 batches
 
-.. note:: When using Trainer(enable_pl_optimizer=True), there is no need to call `.zero_grad()`.
```

Comment:
`enable_pl_optimizer` was removed in #6163.

Reply:
You can just merge master

Reply:
Ah, right, I thought it was missed. Will merge master.